plugins/action/action.h

```diff
@@ -119,6 +119,26 @@ class Action
     Result return_to_launch() const;
 
     /**
+     * @brief Send command to transition the drone to fixedwing.
+     *
+     * Note that this is only for the vtol type.
```

Comment:
Suggest replace both of these points (and update similarly elsewhere) to:

> The associated action will only be executed for VTOL vehicles in multicopter mode. 
> On other vehicles/modes the command will fail with a `Result`.

I assume that the statement is correct - ie that the command will fail with an error?